### ISSUE TITLE
Make the ESLint site look like the JSCS site

### ISSUE THREAD

**Issue by dtracers** [eslint/eslint#6380]
Since jscs and eslint are combining why not have the combination of the website itself.

The jscs website is much easier to navigate and understand where everything is. Can we make the ESLint site easily navigable in a similar fashion as the jscs site?

**Comment by ilyavolodin** [comment]
Could you provide some more details as to why you think jscs site is easier to navigate and understand?

**Comment by hzoo** [comment]
One thing I liked was that the homepage wasn't just some links/navigation but explained more of what the tool did (and I hope the screenshots helped with that). What do we want new users to know about see at first glance?

**Comment by dtracers** [comment]
Yes I can!

For one on the rules page it is very obvious that the search bar for rules applies directly to rules.  But on eslint the search is a sitewide search.

On eslint the rules are not prominent like they are on jscs.
And it is really difficult to find the github repo from the website on eslint.

On some very personal opinions I feel the jscs website is more approachable when you first get to it.
When you get to jscs you see 4 things that direct your attention: Overview, Rules, Contributing, Changelog

On eslint I felt a little overwhelmed.  Especially because I was just looking for rules.  (I am coming from jscs and was trying to see how progress is going)

On eslint there is the big get started that grabs your vision but trying to find where the rules or how to contribute takes time to find it.  Then there is news stuff and below that there are links to about and rules.  But they take a while to find.

I do want to stress that I do not find that the ESLint is a bad site by any means.  It is very well put together.  I just think the jscs site is friendlier and is approachable.  And I am mainly talking about the landing page.

One other thing I just noticed that I did not realize was a difference but kinda irked me was that ESLint is on the far left on super wide monitors.  But JSCS is centered which is nicer.  (But that is a really tiny nit)

The changelog built into JSCS was also a nice addition.  And made it easy to know what changed without having to go to the github.

I really hope these help.  I do not mean this as a bashing session but more of trying to get the best of both worlds including the websites.  (btw the demo in the browser is awesome!)
